packages/react-ui/src/app/features/builder/flow-canvas/context-menu/utils.ts

```diff
@@ -0,0 +1 @@
+export const isProtocolHttps = window.location.protocol === 'https:';
```

Comment:
Paste should be enabled in localhost where it's a secure context:
https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts#when_is_a_context_considered_secure